emperor/format.py

```diff
@@ -101,7 +101,7 @@ def format_pcoa_to_js(header, coords, eigvals, pct_var, custom_axes=[],
 
     # write the values for all the ellipses
     js_pcoa_string += '\nvar g_ellipsesDimensions = new Array();\n'
-    if coords_low != None and coords_high != None:
+    if isinstance(coords_low, ndarray) and isinstance(coords_high, ndarray):
```

Comment:
Yeah, that does it, I'm having a slow day today ... as it turns out.

On (Jan-23-15|14:47), Jai Ram Rideout wrote:
> > @@ -101,7 +101,7 @@ def format_pcoa_to_js(header, coords, eigvals, pct_var, custom_axes=[],
> >  
> >      # write the values for all the ellipses
> >      js_pcoa_string += '\nvar g_ellipsesDimensions = new Array();\n'
> > -    if coords_low != None and coords_high != None:
> > +    if isinstance(coords_low, ndarray) and isinstance(coords_high, ndarray):
> 
> Would it be simpler to test:
> 
> ```python
> if coords_low is not None and coords_high is not None:
> ```
> 
> ?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/331/files#r23486351